packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -90,75 +148,28 @@ export function TransactionsTableRowAction({
       return null
     }
 
-    return (
-      <Tooltip
-        show={!currentChainIsValid}
-        wrapperClassName=""
-        content={
-          <span>
-            {`Please switch to ${getNetworkName(
-              tx.destinationChainId
-            )} to claim your ${tx.isWithdrawal ? 'withdrawal' : 'deposit'}.`}
-          </span>
-        }
+    return isClaiming || isClaimingCctp ? (
+      <span className="my-2 animate-pulse text-xs">Claiming...</span>
+    ) : (
+      <Button
+        variant="primary"
+        className="w-14 rounded bg-green-400 p-2 text-xs text-black"
+        onClick={handleClaim}
       >
-        <Button
-          variant="primary"
-          loading={isClaiming || isClaimingCctp}
-          disabled={isClaimButtonDisabled}
-          className={twMerge(!currentChainIsValid && 'p-2 py-4 text-xs')}
-          onClick={async () => {
-            try {
-              if (!currentChainIsValid) {
-                return switchNetwork?.(tx.destinationChainId)
-              }
-
-              if (tx.isCctp) {
-                return await claimCctp()
-              } else {
-                return await claim(tx)
-              }
-            } catch (error: any) {
-              if (isUserRejectedError(error)) {
-                return
-              }
-
-              errorToast(
-                `Can't claim ${
-                  type === 'deposits' ? 'withdrawal' : 'deposit'
-                }: ${error?.message ?? error}`
-              )
-            }
-          }}
-        >
-          {currentChainIsValid ? 'Claim' : 'Switch Network'}
-        </Button>
-      </Tooltip>
+        Claim
+      </Button>
     )
   }
 
   if (isError) {
-    const isTxOlderThan7Days = dayjs().diff(dayjs(tx.createdAt), 'days') > 7
-
     return (
-      <>
-        {isTxOlderThan7Days ? (
-          // show a dropdown menu with the button
-          <Popover>
-            <Popover.Button>
-              <EllipsisVerticalIcon className="h-6 w-6 cursor-pointer p-1 text-dark" />
-            </Popover.Button>
-            <Popover.Panel
-              className={'absolute top-4 z-50 rounded-md bg-white shadow-lg'}
-            >
-              <GetHelpButton variant="secondary" onClick={getHelpOnError} />
-            </Popover.Panel>
-          </Popover>
-        ) : (
-          // show a normal button outside
-          <GetHelpButton variant="primary" onClick={getHelpOnError} />
-        )}
-      </>
+      <Button
+        variant="primary"
+        className="w-12 rounded bg-white p-2 text-xs text-black"
+        onClick={getHelpOnError}
+      >
+        Get help
+      </Button>
```

Comment:
<img width="925" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/5bf64f59-abdf-41a7-b07b-85e71e121b2d">
Get Help button still truncated on this branch